nix/yazi-unwrapped.nix

```diff
@@ -60,14 +60,27 @@ rustPlatform.buildRustPackage rec {
       magick assets/logo.png -resize "$RES"x"$RES" $out/share/icons/hicolor/"$RES"x"$RES"/apps/yazi.png
     done
 
+    installManPage ${finalAttrs.passthru.srcs.man_src}/yazi{.1,-config.5}
+
     mkdir -p $out/share/applications
     install -m644 assets/yazi.desktop $out/share/applications/
   '';
 
+  passthru.srcs = {
+    man_src = fetchFromGitHub {
+      name = "manpages"; # needed to ensure name is unique
+      owner = "yazi-rs";
+      repo = "manpages";
+      rev = "8950e968f4a1ad0b83d5836ec54a070855068dbf";
```

Comment:
How do we keep this up-to-date?

Reply:
I found that manpages has only one commit and is currently not maintained. So, it's not a big problem to not consider updates here for now.